lualib/rspamadm/secretbox.lua

```diff
@@ -0,0 +1,101 @@
+local util = require 'lua_util'
+local rspamd_util = require 'rspamd_util'
+local argparse = require 'argparse'
+
+local parser = argparse()
+    :name "rspamadm encrypt/decrypt text"
+    :description "Encrypt/Decrypt given text with given key and nonce"
+    :help_description_margin(32)
+    :command_target('command')
+    :require_command(true)
+
+local decrypt = parser:command 'decrypt'
+                      :description 'Decrypt text with given key and nonce'
+
+decrypt:option "-t --text"
+      :description("Encrypted text(Base 64)")
+      :argname("<text>")
+decrypt:option "-k --key"
+      :description("Key used to encrypt text")
+      :argname("<key>")
+decrypt:option "-n --nonce"
+      :description("Nonce used to encrypt text(Base 64)")
+      :argname("<nonce>")
+      :default('nil')
+
+local encrypt = parser:command 'encrypt'
+                      :description 'Encrypt text with given key'
+
+encrypt:option "-t --text"
+      :description("Text to encrypt")
+      :argname("<text>")
+encrypt:option "-k --key"
+      :description("Key to encrypt text")
+      :argname("<key>")
+encrypt:option "-n --nonce"
```

Comment:
Why do we need a nonce for encrypt? It should be a boolean flag: either concat nonce with a string or output two strings.